lib/generate.js

```diff
@@ -35,36 +37,64 @@ function dedentFunc(func) {
     });
 }
 
+function tabulateString(string, indent, filter) {
```

Comment:
Do you ever actually use the `filter` argument? I don't see it being used anywhere except with `id`, which shouldn't change anything.